docs/setup/agent_configuration.mdx

```diff
@@ -2,3 +2,27 @@
 title: DSK Agent Configuration
 sidebar_position: 5
 ---
+
+After successfully linking an Agent using the [**Agent Companion**](./agent_companion.mdx) it can be configured.
+The recommended option is to configure the Agent via the Dataspace Admin (DSA).
+
+
+## Configure Agent
```

Comment:
It would be good if we describe how the `Apply` and `Apply Configuration` buttons play together, since users had difficulties in the past.